packages/trpc/server/routers/loggedInViewer/meStats.handler.ts

```diff
@@ -0,0 +1,55 @@
+import type { Session } from "next-auth";
+
+import prisma from "@calcom/prisma";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+type MeStatsOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+    session: Session;
+  };
+};
+
+export const meStatsHandler = async ({ ctx }: MeStatsOptions) => {
+  const { user: sessionUser } = ctx;
+
+  const additionalUserInfo = await prisma.user.findFirst({
+    where: {
+      id: sessionUser.id,
+    },
+    select: {
+      bookings: {
+        select: { id: true },
+      },
+      selectedCalendars: true,
+      teams: {
+        select: {
+          team: {
+            select: {
+              id: true,
+              eventTypes: true,
+            },
+          },
+        },
+      },
+      eventTypes: {
+        select: { id: true },
+      },
+    },
+  });
+  let sumOfTeamEventTypes = 0;
+  for (const team of additionalUserInfo?.teams || []) {
+    for (const _eventType of team.team.eventTypes) {
+      sumOfTeamEventTypes++;
+    }
+  }
```

Comment:
const sumOfTeamEventTypes = additionalUserInfo?.teams.reduce(
    (sum, team) => sum + team.team.eventTypes.length,
    0
  );

Reply:
I just moved this code. Nice find. Will update.